src/Workleap.Extensions.Configuration.Secrets/TokenCredentialProvider.cs

```diff
@@ -47,9 +57,15 @@ private static TokenCredential GetTokenCredentialInternal(string environmentName
     private static TokenCredential GetAzureCliCompatibleTokenCredential()
     {
         // Azure CLI does not work when Fiddler is active so we need to use an interactive authentication method instead
-        // When Fiddler is not active, we try to use AzureCliCredential because it's way faster than DefaultAzureCredential on startup (instantaneous VS several seconds)
-        return FiddlerProxyDetector.IsFiddlerActive()
-            ? new CachedInteractiveBrowserCredential()
+        if (FiddlerProxyDetector.IsFiddlerActive())
```

Comment:
An if, followed by ternary operation : would it be more readable to have a classic if-else to it's uniform?